sdks/python/apache_beam/io/bigquery_test.py

```diff
@@ -40,6 +40,27 @@ def test_row_as_dict(self):
     test_value = {'s': 'abc', 'i': 123, 'f': 123.456, 'b': True}
     self.assertEqual(test_value, coder.decode(coder.encode(test_value)))
 
+  def test_invalid_json_nan(self):
+    with self.assertRaises(ValueError) as exn:
```

Comment:
That's done here self.assertTrue(bigquery.JSON_COMPLIANCE_ERROR in exn.exception.message). Do you want to make sure that the base error is what we expect? We only catch the ValueError, I could also inspect the string of the ValueError in bigquery.py to make sure I only raise when it matches that.